HR-Monitor/src/Main/Configuration.java

```diff
@@ -4,7 +4,7 @@
 import java.nio.file.Paths;
 
 public class Configuration {
-    public static final String BES_URI = "http://localhost:9090/api/HRController/";
+    public static final String Backend_Server_URL = ""; //  Http or Https
```

Comment:
should be https. this change is there in my PR.

Reply:
Users can run the HRM also in HTTP! 
personal configuration of the users!